### DISCUSSION TITLE
[Skia.Wpf] MainPage matches more then one interface member.

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#9465] [Q&A] [answered]
I created empty Uno project.
When I'm trying to build Wpf Skia project I have following result:
![зображення](https://user-images.githubusercontent.com/69314237/184134847-fb4ef7e0-75bb-49e8-b62c-5ef8dd210c11.png)

![зображення](https://user-images.githubusercontent.com/69314237/184134938-ad108ddb-8349-4d99-bcee-37fbf16c60fb.png)

I have 3 600 errors and I don't know which one I sould to send, but I hope that warnings will explain this problem:
```
Severity	Code	Description	Project	File	Line	Suppression State
Warning	CS0473	Explicit interface implementation 'MainPage.MainPage_Bindings.MainPage.IMainPage_Bindings.Initialize()' matches more than one interface member. Which interface member is actually chosen is implementation-dependent. Consider using a non-explicit implementation instead.	Zhyttya.UI.Uno.Skia.Wpf	L:\zhyttya\src\Zhyttya.UI.Uno\Zhyttya.UI.Uno.Skia.Wpf\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator\MainPage_fc7b9b5ccd6ce94c36b89f05e8f576a3.cs	138	Active
```

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. You're facing https://github.com/unoplatform/uno/issues/9380#issuecomment-1204398921, see the post for a workaround.
